pkg/leakybucket/blackhole.go

```diff
@@ -34,8 +34,8 @@ func NewBlackhole(g *BucketFactory) (*Blackhole, error) {
 	}, nil
 }
 
-func (bl *Blackhole) OnBucketOverflow(b *BucketFactory) func(*Leaky, types.SignalOccurence, *Queue) (types.SignalOccurence, *Queue) {
-	return func(l *Leaky, s types.SignalOccurence, q *Queue) (types.SignalOccurence, *Queue) {
+func (bl *Blackhole) OnBucketOverflow(b *BucketFactory) func(*Leaky, types.Alert, *Queue) (types.Alert, *Queue) {
+	return func(l *Leaky, s types.Alert, q *Queue) (types.Alert, *Queue) {
```

Comment:
can you please rename the ones letter variable by more understandable name please ?